pex/pex_rules.bzl

```diff
@@ -156,10 +156,15 @@ def _gen_manifest(py, runfiles):
         ),
     )
 
+  # All .so files that end with .so, .so.#, .so.#.#, etc.
```

Comment:
Will these ever be `.dylib` (macos) or `.dll` (windows), or do python libs always end up named .so regardless of platform?

Reply:
Done, though since we don't have platform information (yet? see https://github.com/bazelbuild/bazel/issues/2045), I had to add all possibilities. This means that if you, for instance, create a PEX on linux that's going to ship a pyd to a Windows machine, that we're going to extract the pyd early for no reason. Oops!